dev/setup-single-cluster

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I like the idea of these `setup-{single,multi}-cluster`, which simplifies the dev `README` and offers flexibility for users to override the `envrc` with their own variables. It is nicely explained in the `README`, too :)

Reply:
Thank you very much!